integration_tests/client/datatype/test_data_generation.py

```diff
@@ -322,17 +321,17 @@ def test_generate_segmentation_data(
     ), "Number of images doesn't match the test input"
 
     for image in dataset.get_datums():
-        uid = image.uid
+        uid = image.get_uid()
```

Comment:
nit: could we change `.get_uid` to `.uid` (related to my other comment above)?

Reply:
The problem is that uid is reserved for the `String` object which requires a `get_value` call to unpack itself.